README.md

```diff
@@ -108,6 +112,119 @@ async onRequestInit({ requestInit }) {
 }
 ```
 
+If you're accepting webhooks from Clerk, you will need to validate the incoming webhook signature. This can be done like this:
```

Comment:
This should be under a new heading and should mention which Svix package needs to be installed.